extensions/hibernate-orm/deployment/src/test/java/io/quarkus/hibernate/orm/config/DevUISchemaManagementStrategyTest.java

```diff
@@ -0,0 +1,25 @@
+package io.quarkus.hibernate.orm.config;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.eclipse.microprofile.config.ConfigProvider;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkus.test.QuarkusUnitTest;
+
+public class DevUISchemaManagementStrategyTest {
+
+    // A simple runner like this will trigger Dev UI code
```

Comment:
Dev UI is not the same thing as dev services. I think you mean Dev services here.